services/venice-router/src/main/java/com/linkedin/venice/router/api/VeniceHostHealth.java

```diff
@@ -74,9 +76,24 @@ public void setHostAsHealthy(Instance hostname) {
       unhealthyHosts.remove(identifier);
```

Comment:
Minor nitpick... there is no need to do `contains` followed by `remove`... we can just `remove` directly, and the return value is what was removed, if any. So the code can be:

```
if (unhealthyHosts.remove(identifier) != null) {
  // LOGGER ...
}
```

Reply:
Sure, I can make the change.